### ISSUE TITLE
Localise

### ISSUE THREAD

**Issue by bodhisattwawiki** [lokal-profil/credit-my-cc#3]
Hi, it would be great if the tool is localised in other languages.

**Comment by Ainali** [comment]
Agreed. As soon as it is translated to English (in #1 ) we are a good bit on the way.

**Comment by lokal-profil** [comment]
The reason I haven't merged #7 and #9 is because the underlying functionality to switch between languages is still missing. Without having the interface translatable I'm not sure about the value of having form letters in other languages.

**Comment by Ainali** [comment]
Ping @lokal-profil, I think developing this would be a good fit for the [NGI Zero Commons Fund](https://nlnet.nl/commonsfund/). Do you think localization could be prioritized if it got funded? If so, I would be happy to start writing on an application (next deadline is 1 June).

**Comment by lokal-profil** [comment]
@Ainali Sorry for missing this ping. In general. WMSE is currently not doing any development on this tool. If we wanted it to be developed further I think it would probably be good to redo from scratch (to get proper localisation support and mobile support). If someone would want to take that on I'd be happy to support in an advisory role.
